lib/Middlewares/Third_Party/littlefs/CMakeLists.txt

```diff
@@ -1,5 +1,15 @@
 add_library(littlefs lfs.c lfs_util.c)
 
+# Required printfs for debug print is not available, therefore disable all prints
```

Comment:
we may want to replace the LFS's debug/warn/error output with our logging subsystem - to be considered in the future

Reply:
Yes, we can replace this with something like
```
  PROPERTY COMPILE_DEFINITIONS
            LFS_DEBUG print_to_serial("LFS DEBUG: " fmt, __VA_ARGS__)
```